src/services/meta_transaction_service.ts

```diff
@@ -137,7 +137,8 @@ export class MetaTransactionService {
                     taker: params.takerAddress,
                     quoteReportSources: quote.extendedQuoteReportSources,
                     submissionBy: 'gaslessSwapAmm',
-                    decodedUniqueId: params.quoteUniqueId ? params.quoteUniqueId : quote.decodedUniqueId,
+                    decodedUniqueId: quote.decodedUniqueId,
+                    ammQuoteUniqueId: params.quoteUniqueId,
```

Comment:
don't love that there's a name change here. Can we just call this field `quoteUniqueId`?

Reply:
The name of the field can't be changed since this object would be produced to quote report Kafka topic.

Maybe we can change the query param name in `params` but I do think the current name in query param `quoteUniqueId` is accurate since it's the `decodedUniqueId` for corresponding gasless swap RFQ quote report

Reply:
Gotcha - understood. Can we leave a comment on why there's a name change here?